uploader.json

```diff
@@ -3335,6 +3335,11 @@
             "symbolic_name": "amq_broker",
             "pattern": []
         },
+        {
+            "file": "/boot/()config-.*",
```

Comment:
I think it should be `"glob"`.

Reply:
Hi @xiangce  https://github.com/RedHatInsights/insights-core-assets/pull/283 I have changed as per @bfahr 's comment

Reply:
@vishwanathjadhav that is correct.  @xiangce the v1 client does not support globs, and only supports regex in the basename of the path, so that is the correct way to specify the regex in the uploader.json.

Reply:
@vishwanathjadhav this entry needs to be moved to the "files" section of the json, it is in the "globs" section of the json.